traffic_ops/testing/api/v14/regexrevalidatedotconfig_test.go

```diff
@@ -0,0 +1,40 @@
+package v14
+
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestRegexRevalidateDotConfig(t *testing.T) {
+	WithObjs(t, []TCObj{CDNs, Types, Tenants, Parameters, Profiles, Statuses, Divisions, Regions, PhysLocations, CacheGroups, Servers, DeliveryServices}, func() {
+		CreateTestJobs(t)
+		GetTestRegexRevalidateDotConfig(t)
+	})
+}
+
+func GetTestRegexRevalidateDotConfig(t *testing.T) {
+	cdnName := "cdn1"
+
+	cfg, _, err := TOSession.GetATSCDNConfigByName(cdnName, "regex_revalidate.config")
```

Comment:
```
./regexrevalidatedotconfig_test.go:30:26: TOSession.GetATSCDNConfigByName undefined (type *client.Session has no field or method GetATSCDNConfigByName)
```
It looks like you forgot to add and commit that new client method.

Reply:
Fixed.